dataforest/utils/plots_config.py

```diff
@@ -0,0 +1,258 @@
+from copy import deepcopy
+import json
+from collections import OrderedDict
+
```

Comment:
Return value type hinting + brief description of what the keys/values represent (when the function returns a dict) would hugely clarify this file

Reply:
Thanks, added some docstrings :)